contracts/attribute/OwnableStub.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.13;
+
+import "./Ownable.sol";
```

Comment:
We're trying to move away from implicit imports; recommend `import { Ownable } from "./Ownable.sol";`

Reply:
updated in [06c0b18](https://github.com/equilibria-xyz/root/pull/110/commits/06c0b18b012433e0af3f3210abdcae517f7ef11b)